src/views/Map/Layouts/Mobile/Mobile.tsx

```diff
@@ -9,6 +9,7 @@ export function MobileLayout({ firstColumn, mapColumn, footer, drawToolbox }) {
         <div className={s.mapSpaceRight}>{mapColumn}</div>
       </div>
       <div className={s.footerWrap}>{footer}</div>
+      <div className={s.fakeFooter}></div>
```

Comment:
How about giving it a more meaningful name?
A "fake footer" doesn't say why this needed, and what doing, only that it's fake.